fanuc_lrmate200id_moveit_config/config/controllers.yaml

```diff
@@ -1,5 +1,5 @@
 controller_list:
-  - name: ""
-    action_ns: joint_trajectory_action
+  - name: joint_trajectory_controller
+    action_ns: follow_joint_trajectory
     type: FollowJointTrajectory
     joints: [joint_1, joint_2, joint_3, joint_4, joint_5, joint_6]
```

Comment:
We cannot change this file, as it is used with the default `moveit_planning_execution.launch`.

You'd have to remap the AS topics to the 'ros-industrial specs'. See [abb_experimental/abb_irb120_gazebo/launch/irb120_gazebo.launch](https://github.com/ros-industrial/abb_experimental/blob/91de854692a8166014f8ec8e3b9565299b79c27d/abb_irb120_gazebo/launch/irb120_gazebo.launch#L5-L7) for an example.